.github/workflows/test_fast.yml

```diff
@@ -48,37 +48,10 @@ jobs:
       - name: Configure git  # Needed by the odd test
         uses: cylc/release-actions/configure-git@v1
 
-      - name: Check changelog
-        if: startsWith(matrix.os, 'ubuntu')
-        run: towncrier build --draft
-
-      - name: Style
-        if: startsWith(matrix.os, 'ubuntu')
-        run: |
-          flake8
-          etc/bin/shellchecker
-
-      # note: exclude python 3.10+ from mypy checks as these produce false
-      # positives in installed libraries for python 3.7
-      - name: Typing
-        if: startsWith(matrix.os, 'ubuntu') && startsWith(matrix.python-version, 3.9)
-        run: mypy
-
-      - name: Doctests
-        timeout-minutes: 4
-        run: |
-          pytest cylc/flow
-
       - name: Unit Tests
-        timeout-minutes: 4
-        run: |
-          pytest tests/unit
-
-      - name: Bandit
-        if: ${{ matrix.python-version == '3.7' }}
-        # https://github.com/PyCQA/bandit/issues/658
+        timeout-minutes: 5
         run: |
-          bandit -r --ini .bandit cylc/flow
+          pytest cylc/flow tests/unit
```

Comment:
If you're consolidating these into one step, it would be good to set `-q` for quiet in `PYTEST_ADDOPTS`, to prevent us having to scroll thousands of log lines to get to the test failures. Note that when `CI=true` Pytest always outputs full diffs regardless of the verbosity option.

Also `-l` for displaying the value of locals might be handy.

Reply:
I don't see combining these sections as a reason for reducing verbosity, better to have the required information than to hide it?

Reply:
I'm talking about hiding these lines without reducing the verbosity of failed assertion output

![image](https://github.com/cylc/cylc-flow/assets/61982285/ba35480b-6d66-4c84-a9d4-a2c578199388)

Reply:
I don't really see a problem with this and seeing the test output is helpful for debugging timeouts.